packages/gatsby-plugin-benchmark-reporting/src/gatsby-node.js

```diff
@@ -51,7 +51,7 @@ class BenchMeta {
 
     return {
       time: this.localTime,
-      sessionId: uuidv4(),
+      sessionId: process.gatsbyTelemetrySessionId,
```

Comment:
In some edge cases the `process.gatsbyTelemetrySessionId` might not be defined, so let's add some fallback to at least differentiate between sessions.

```suggestion
      sessionId: process.gatsbyTelemetrySessionId || uuidv4(),
```